Sponsors.md

```diff
@@ -1,90 +1,80 @@
-* 5 swag packs will be awarded at each meetup
-* Each pack costs 20 EUR
-* 100 EUR buys us 5 packs - i.e. awareness for you for ONE meetup. 
-* 200 EUR buys us 10 packs - i.e. awareness for you for TWO meetups. 
-* 400 EUR buys us 20 packs - i.e. awareness for you for FOUR meetups. 
+## Other ways to sponsor
 
-The list below describes the awareness benefits that our branding sponsor receives in return:
-* leaflets (can be left on table at entrance),
-* “Thank you” message by organizers during Meetup (not by company),
-* “Thank you” message during Swag Pack draw
-* “Thank you” mention in facebook page post.
+We provide a few other ways to sponsor DevStaff outside OpenCollective, that provides the advantage to a legal entity (e.g., a company) to be directly invoiced. Your contribution will be made aware to our community (e.g., during meetups), but note that it cannot be listed in our OpenCollective page.
 
-Note: the invoices will be issued directly to the sponsoring company by our own suppliers. 
+### Swag Sponsor
+
+Help the community members wear the DevStaff community name with pride! With this option, you can sponsor the purchase of promotional material with the DevStaff brand, in return for awareness of your own brand.  (Please note we do not currently offer a co-branding option, see below for awareness benefits). All material will be made available to community and org team members.
 
-## DevStaff Laptop Stickers Sponsor
-No developer laptop is complete without laptop stickers. A “badge of honor” for many, a hipster trend for some… but the reality is that laptop stickers are highly sought after! 
+No developer is complete without a dev-friendly cool looking t-shirt or laptop stickers. A "badge of honor" for many, a hipster trend for some... but the reality is that dev t-shirts and laptop stickers are highly sought after! Each swag pack may include a dev-friendly cool looking t-shirt, and/or "Raki" / "Tsikoudia" shot glasses, and/or laptop stickers, and/or any other item that can fit in the budget. 
 
-Estimated cost: ~300 EUR for 500 stickers
+Swag packs may be given to lucky attendees (e.g., through a lottery that can take place at the meetup), or to speakers, or to community members. Your company will be publicly thanked for making this possible. 
 
-The list below describes the awareness benefits that our stickers sponsor receives in return:
-* leaflets (can be left on table at entrance),
-* “Thank you” message by organizers during Meetup (not by company),
-* “Thank you” message in every meetup, as long as we still have stickers to give away.
-* “Thank you” mention in facebook page post.
+Get in contact with us (at [info@devstaff.gr](mailto:info@devstaff.gr?subject:Swag%20Sponsorship)) to discuss swag pack opportunities. Please note that depending on the amount, we will publicly thank you in a number of next meetups. For example:
+
+* Around 100 EUR on swag packs - i.e. awareness for you for ONE meetup. 
```

Comment:
Shouldn't we align the prices according to the latest updates of open collective sponsorship options?

Reply:
Yes I agree on this.

Reply:
Makes sense, done!